### ISSUE TITLE
metadat - whats the difference between "title" and "distribution:title" ?

### ISSUE THREAD

**Issue by stevieflow** [ThreeSixtyGiving/standard#254]
In practice, I think these are used in exactly the same way.  

eg:

> "title":"Arcadia Fund grants awarded to April 2018",
>              },
>              "distribution":[ {
>                 "title":"Arcadia Fund grants awarded to April 2018"}
> 

Is there a distinct way we should be using these, though?

@KDuerden - when inputting via Salesforce, do you utilise these differently?

Im also not sure if we provide for distribution:title in the metadata package @kindly 

Relates to https://github.com/ThreeSixtyGiving/registry/issues/27

**Comment by stevieflow** [comment]
Thanks

@kindly what would be an example use case where the two ``Title`` fields have a difference?  Could it be when they output the same data, but in different formats, for example?

**Comment by kindly** [comment]
@stevieflow yes that is use case.   Sometimes distributions also are used to partition data too i.e a to f or 2015-2016.   
The dataset:title would then just express what that partition or format was i.e "JSON File" and not repeat the dataset title.  However, there is no good guidance in dcat what good titles are.
